pkg/util/filesystem/util_windows.go

```diff
@@ -85,3 +87,10 @@ func IsUnixDomainSocket(filePath string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsAbs returns wether the given path is absolute or not.
```

Comment:
This needs a big warning on it that it isn't safe to use for API values which will propagate across systems (e.g. REST API values that get validated on Unix, persisted, then consumed by Windows, etc)

Reply:
+1 Jordan. Let's do this @mxpv

Reply:
Updated in cefe0b2ed04806186ecc671474bf2dbece4857ae